app/controllers/concerns/images_utility.rb

```diff
@@ -0,0 +1,40 @@
+module ImagesUtility
+
+  private
+
+  def download_image(type, width, image_html)
```

Comment:
You probably should remove the `type` argument as it is not needed anymore.

Reply:
This refactoring definitely improves the readability of the code.

Reply:
`type` is still needed
```
  def download_image(type, width, image_html)
    kit = build_kit(image_html, "#{**type**.tr('_', '-')}.css", width)
    send_data(kit.to_jpg, type: 'image/jpg', filename: "#{**type**}.jpeg")
  end
```

Reply:
oops ... you're right.  I scanned too quickly.